Octokit/Http/Connection.cs

```diff
@@ -498,14 +498,14 @@ public async Task<HttpStatusCode> Put(Uri uri)
         /// Performs an asynchronous HTTP PUT request that expects an empty response.
         /// </summary>
         /// <param name="uri">URI endpoint to send request to</param>
-        /// <param name="accepts">Specifies accepted response media types.</param>
+        /// <param name="body">The object to serialize as the body of the request</param>
         /// <returns>The returned <seealso cref="HttpStatusCode"/></returns>
-        public async Task<HttpStatusCode> Put(Uri uri, string accepts)
+        public async Task<HttpStatusCode> Put(Uri uri, object body)
```

Comment:
I've double-checked with the current main branch, and this method has only one reference to IConnection.cs.

Since I needed to pass an object, initially I added the object body as an additional parameter. Then I saw that the string accepts is not being passed from anywhere in the codebase, so I decided to remove that argument.

Reply:
Understood, thank you.